content/pt-br/docs/reference/glossary/garbage-collection.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Coleta de lixo
+id: garbage-collection
+date: 2021-07-07
+full_link: /docs/concepts/architecture/garbage-collection/
+short_description: >
+  Um termo coletivo para os vários mecanismos que o Kubernetes usa para limpar os recursos do cluster.
+
+aka: 
+tags:
+- fundamental
+- operation
+---
+
+Coleta de lixo (Garbage collection) é um termo coletivo para os vários mecanismos que o Kubernetes usa para limpar os recursos do cluster.
+
+<!--more-->
+
+O Kubernetes usa a coleta de lixo para limpar recursos como 
```

Comment:
The main meaning is a Garbage Collector, mechanism to remove all the unused resources.
Please, apply the previous change here as well.
Thanks